src/OrchardCore.Modules/OrchardCore.Users/Startup.cs

```diff
@@ -118,7 +118,12 @@ public override void ConfigureServices(IServiceCollection services)
 
             // Adds the default token providers used to generate tokens for reset passwords, change email
             // and change telephone number operations, and for two factor authentication token generation.
-            services.AddIdentity<IUser, IRole>().AddDefaultTokenProviders();
+            services.AddIdentity<IUser, IRole>(opt =>
+            {
+                opt.Lockout.AllowedForNewUsers = true;
+                opt.Lockout.DefaultLockoutTimeSpan = TimeSpan.FromMinutes(2);
+                opt.Lockout.MaxFailedAccessAttempts = 3;
```

Comment:
Why these options can't come from the settings?

Reply:
This is the goal @hishamco .
I just pushed the PR to show what I tried so far.
No need to review right now.

I am working on it with @Skrypt .